src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+    @Override
```

Comment:
Please add a line break/new line between lines 4 and 5, and remove the new line on line 9